conductr_cli/host.py

```diff
@@ -43,4 +44,55 @@ def with_docker_machine():
 
 
 def loopback_device_name():
-    return 'lo' if platform.system().lower() == 'linux' else 'lo0'
+    return 'lo' if is_linux() else 'lo0'
+
+
+def is_linux():
+    return platform.system().lower() == 'linux'
+
+
+def is_osx():
+    return platform.system().lower() == 'darwin'
+
+
+def can_bind(ip_addr, port):
+    socket_af = socket.AF_INET if ip_addr.version == 4 else socket.AF_INET6
+    s = socket.socket(socket_af, socket.SOCK_STREAM)
+
+    result = False
+    try:
+        s.bind((ip_addr.exploded, port))
+        result = True
+    except OSError:
+        pass
+    finally:
+        s.close()
+
+    return result
+
+
+def addr_alias_setup_instructions(addrs, subnet_mask):
+    if_name = loopback_device_name()
+
+    if is_linux():
+        commands = ['sudo ifconfig {}:{} {} netmask {} up'.format(if_name, idx, addr.exploded, subnet_mask)
+                    for idx, addr in enumerate(addrs)]
+
+        return 'Run the following commands to create address alias on your machine:\n' \
```

Comment:
Use singular or plural depending on the `commands` size:
- `commands == 1`: "Run the following command to create an address alias on your machine"
- `commands > 1`: "Run the following commands to create address aliases on your machine"